app/src/About.vue

```diff
@@ -0,0 +1,146 @@
+<template>
+  <div class="about-us">
+    <h1>Welcome to Our Company</h1>
+
+    <section class="company-intro">
+      <h1>Who We Are</h1>
+      <p>
+        We are a passionate team dedicated to providing innovative solutions
+        and excellent services. Our journey started with a vision to make a
+        positive impact in the world.
+      </p>
+    </section>
+
+    <h2>Our Projects</h2>
+    <ScrollableCards :items="items" />
+
+    <section class="timeline">
+      <h2>Our Journey</h2>
+      <Timeline />
+    </section>
+
+    <h1>Let's build our future together!</h1>
+    <router-link to="/contact">
+      <Button><p>Contact Us</p></Button>
```

Comment:
do you really need `<p>` inside `<Button>`?

Reply:
Thanks, just removed it in my new commit!